### PR TITLE
Annotate container image with github source

### PR DESCRIPTION
**Pull request by gberche-orange**
**What this PR does / why we need it**:

Fixes https://github.com/kudobuilder/kuttl/issues/390

### CODE REVIEW

**Comment by kensipe** [comment]
This seems pretty straight forward... @gberche-orange could you sign your commit for DCO?

**Comment by gberche-orange** [comment]
Thanks @kensipe 

Sorry for the lack of DCO signoff from my commit originated from github web ui. This is now fixed.

ps: I've rebased the commit (`git rebase --signoff HEAD~1`) and configured my works to require sign-offs for
*  next github web commits ( see related https://github.blog/changelog/2022-06-08-admins-can-require-sign-off-on-web-based-commits/) 
* next intellij-based commits (see https://www.jetbrains.com/help/idea/commit-changes-dialog.html#before_commit)

So, hopefully my next commits won't be missing DCO again.

**Comment by gberche-orange** [comment]
It might be possible that if the `/kudobuilder/kuttl` repo or `/kudobuilder` org enables the `Require contributors to sign off on web-based commits` option, then any new fork would automatically inherit from this option, and offer by default to sign off commits made through GH web UI.

**Comment by kensipe** [comment]
thanks for patience... I've taken to long to get to this.  I will review tomorrow this and the other PRs... really appreciate the contributions!

File: Dockerfile [hunk 1]
```diff
@@ -27,6 +27,8 @@ RUN make cli
 # release image with kubectl + kuttl
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 
+LABEL org.opencontainers.image.source="https://github.com/kudobuilder/kuttl/"
```
Comment (erikgb) [inline Dockerfile:30]:
```suggestion
LABEL org.opencontainers.image.source="https://github.com/kudobuilder/kuttl"
```
I am not sure what's correct, but it seems to me that it's more common to not include a trailing slash when using this label.

Comment (gberche-orange) [reply Dockerfile]: thanks @erikgb !

**Review by erikgb — APPROVED** [review]
\lgtm

**Comment by porridge** [comment]
Oh no, I clicked the button to refresh the branch and now the DCO check fails! :sob:
@gberche-orange would it be too much if I asked you to please rebase your commit?

**Review by porridge — APPROVED** [review]

**Comment by gberche-orange** [comment]
@porridge sure, np, rebased and squashed in https://github.com/kudobuilder/kuttl/commit/ce0a883c448171bedf6b82a620d0cb9086b9103b

**Pull request merged**